pyproject.toml

```diff
@@ -39,3 +79,6 @@ source = [
     ".tox/*/lib/python*/site-packages/",
     ".tox/pypy*/site-packages/",
 ]
+
+[tool.setuptools.packages.find]
+where = ["src"]
```

Comment:
With our `src` layout we don't need these lines.  setuptools will find the packages itself.  In the script for moving to PEP 420 native namespaces we also removed similar lines from `setup.py`.

```suggestion
```